packages/smart/test/bpool-test.ts

```diff
@@ -1,4 +1,4 @@
-import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signer-with-address";
```

Comment:
Why this change? I'd like to avoid depending on `dist` directly.

Reply:
I've had to alias `@nomiclabs/hardhat-ethers` to `hardhat-deploy-ethers` for compatibility reasons (https://github.com/wighawag/hardhat-deploy#installation) and was getting ts errors unless I did this unfortunately.